sdk/file_sdk/UnitTests/gtest/SConscript

```diff
@@ -0,0 +1,29 @@
+Import('file_env target_name')
+
+Import("""
+    googlemock_lib
+    googletest_lib
+""")
+
+src_files = ['unittest_main.cpp']
+
+'''
+src_files += Glob('#src/upe/api_impl/test/*.cpp')
+src_files += Glob('#src/upe/base/test/*.cpp')
+src_files += Glob('#src/upe/common/test/*.cpp')
+src_files += Glob('#src/upe/core/test/*.cpp')
+src_files += Glob('#src/upe/core/parser/test/*.cpp')
+src_files += Glob('#src/upe/store/test/*.cpp')
+src_files += Glob('#src/shared/telemetry/test/*.cpp')
```

Comment:
why include things that are not there and will probably will be in other path?

Reply:
these are commented out... will remove it

---
In reply to: [139422602](https://github.com/AzureAD/rms-sdk-for-cpp/pull/161#discussion_r139422602) [](ancestors = 139422602)